src/scss/components/radio/_index.scss

```diff
@@ -0,0 +1,264 @@
+hx-radio-control {
+  > label > hx-radio {
+    background-color: $gray-0;
+    border: 1px solid currentColor;
+    color: $gray-500;
+  }
+
+  > p {
+    //@include HelpText();
+    color: $gray-750;
+    font-size: 0.75rem;
+  }
+
+  > input[type="radio"] {
+    color: $gray-500;
+
+    &::-ms-check {
+      background-color: $gray-0;
+      border: 1px solid currentColor;
+      color: inherit;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+
+      ~ label > hx-radio {
+        box-shadow: $focus-glow;
+      }
+    }
+
+    &:hover {
+      color: $cyan-500;
+
+      &::-ms-check {
+        background-color: $cyan-50;
+        color: $cyan-500;
+      }
+    }
+
+    &:checked {
+      color: $cyan-900;
+
+      ~ label > hx-radio {
+        border: 1px solid currentColor;
+        color: $cyan-900;
+        position: relative;
+
+        &::after {
+          background-color: currentColor;
+          border-radius: 1rem;
+          content: "";
+          height: 0.5rem;
+          left: 50%;
+          position: absolute;
+          top: 50%;
+          transform: translate(-50%, -50%);
+          width: 0.5rem;
+        }
+      }
+    }
+
+    &:valid ~ label > hx-radio:hover {
+      background-color: $cyan-50;
+      border: 1px solid currentColor;
+      color: $cyan-500;
+    }
+  }
+}
+
+// INVALID STATE
+hx-radio-set[hx-dirty] input[type="radio"]:enabled:invalid,
+hx-radio-set.hxInvalid input[type="radio"]:enabled {
+  color: $red-900;
+
+  &::-ms-check {
+    border-width: 2px;
+  }
+
+  ~ label > hx-radio {
+    border: 2px solid currentColor;
+    color: $red-900;
+  }
+
+  &:focus {
+    box-shadow: $focus-glow-invalid;
+
+    ~ label > hx-radio {
+      box-shadow: $focus-glow-invalid;
+    }
+  }
+
+  // native/ie11 styles
+  &:hover::-ms-check {
+    background-color: $red-100;
+    border: 2px solid currentColor;
+    color: $red-900;
+  }
+
+  // facade styles
+  ~ label > hx-radio:hover {
+    background-color: $red-100;
+    border: 2px solid currentColor;
+    color: $red-900;
+  }
+}
+
+// DISABLED STATE
+hx-radio-control > input[type="radio"]:disabled {
+  color: $gray-500;
+  cursor: not-allowed;
+
+  // ie11 styles
+  &::-ms-check {
+    background-color: $gray-100;
+    border: 1px solid currentColor;
+    color: $gray-500;
+  }
+
+  &:focus {
+    box-shadow: $focus-glow-disabled;
+  }
+
+  &:hover {
+    &::-ms-check {
+      background-color: $gray-100;
+      border: 1px solid currentColor;
+      color: $gray-500;
+    }
+  }
+
+  // facade styles
+  ~ label {
+    color: $gray-700;
+
+    > hx-radio {
+      border: 1px solid currentColor;
+      color: $gray-500;
+      cursor: not-allowed;
+
+      &,
+      &:hover {
+        background-color: $gray-100;
+        border: 1px solid currentColor;
+        color: $gray-500;
+      }
+    }
+  }
+}
+
+@supports (--modern: true) {
+  hx-radio-control {
+    position: relative;
+
+    > input[type="radio"] {
+      //@mixin hideInput{}
```

Comment:
Please use the SCSS mixin to hide the input. This is the reason why there are two radios showing up.

```scss
@include hxInput(visuallyHidden);
```

(See https://github.com/HelixDesignSystem/helix-ui/blob/master/src/scss/components/form/mixins/_input.scss#L37)

Reply:
Changes are done ... please review